<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="PW-HAN-063" name="automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test" tests="24" failures="2" timestamp="19 Sep 2019 09:49:48 GMT" time="784.302" errors="0">
  <testcase name="execute" time="18.296" classname="automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test"/>
  <testcase name="execute" time="18.787" classname="automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test"/>
  <testcase name="execute" time="53.453" classname="automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test"/>
  <testcase name="execute" time="38.831" classname="automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test"/>
  <testcase name="execute" time="21.576" classname="automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test"/>
  <testcase name="execute" time="21.182" classname="automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test"/>
  <testcase name="execute" time="34.076" classname="automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test"/>
  <testcase name="execute" time="34.187" classname="automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test"/>
  <testcase name="execute" time="30.085" classname="automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test"/>
  <testcase name="execute" time="33.693" classname="automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test"/>
  <testcase name="execute" time="32.187" classname="automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test"/>
  <testcase name="execute" time="23.142" classname="automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test"/>
  <testcase name="execute" time="38.879" classname="automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test"/>
  <testcase name="execute" time="22.968" classname="automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test"/>
  <testcase name="execute" time="33.651" classname="automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test"/>
  <testcase name="execute" time="18.270" classname="automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test"/>
  <testcase name="execute" time="19.585" classname="automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test"/>
  <testcase name="execute" time="35.460" classname="automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	[Enrolled] expected [] but found [U]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
[Enrolled] expected [] but found [U]
at automation.project3ds.MySoftAssertAll.assertAll(MySoftAssertAll.java:57)
at automation.project3ds.Action.assertResultV1(Action.java:268)
at automation.project3ds.Action.assertResultV1(Action.java:156)
at automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test.execute(BrickHTML_Card1_OFF_Test.java:97)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- execute -->
  <testcase name="execute" time="34.819" classname="automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	[Enrolled] expected [] but found [U]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
[Enrolled] expected [] but found [U]
at automation.project3ds.MySoftAssertAll.assertAll(MySoftAssertAll.java:57)
at automation.project3ds.Action.assertResultV1(Action.java:268)
at automation.project3ds.Action.assertResultV1(Action.java:156)
at automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test.execute(BrickHTML_Card1_OFF_Test.java:97)
at sun.reflect.GeneratedMethodAccessor89.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- execute -->
  <testcase name="execute" time="19.536" classname="automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test"/>
  <testcase name="execute" time="75.606" classname="automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test"/>
  <testcase name="execute" time="53.515" classname="automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test"/>
  <testcase name="execute" time="38.896" classname="automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test"/>
  <testcase name="execute" time="33.622" classname="automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test"/>
</testsuite> <!-- automation.project3ds.oldTest.BrickHTML_Card1_OFF_Test -->
